en/micro-integrator/docs/overview/about-this-release-7.1.0.md

```diff
@@ -0,0 +1,98 @@
+# About this Release
+
+WSO2 Enterprise Integrator (EI) version 7.1.0 is the latest release of the WSO2 EI 7.x family, which provides a powerful solution for integrating systems.
+
+## What's new in this release?
+
+### New in the Micro Integrator
+
+The following features and improvements are available for the <b>Micro Integrator</b> of EI 7.1.0:
+
+-   [Task coordination](../../setup/deployment/deploying_wso2_ei/#cluster-coordination) for distributed deployments.
+-   Capability of viewing and configuring logs without accessing the file system.
```

Comment:
Shall we add via the CLI(link to https://ei.docs.wso2.com/en/latest/micro-integrator/administer-and-observe/using-the-command-line-interface/#mi-log-level), management API(https://ei.docs.wso2.com/en/latest/micro-integrator/administer-and-observe/working-with-management-api/#get-log-level) and the dashboard.